protocol/examples/query/getMessagesById/getMessagesById.json

```diff
@@ -0,0 +1,15 @@
+{
+  "jsonrpc": "2.0",
+  "id": 4,

```

Comment:
Why did you put the field id here? We didn't use it in our JSON Schema

Reply:
It is in the schema on the google docs "id: linkToRawGithub" but I'm not sure if it's actually useful. I also added it because the root schema seems to need it for validation but it still doesn't pass so I'm also a bit confused

Reply:
it's needed for the example to be valid.
https://stackoverflow.com/questions/2210791/json-rpc-what-is-the-id-for

Reply:
Ah okay, I didn't understand that. But then, why is it missing in the Broadcast JSON Schema?

Reply:
If I understood the stackoverflow answers properly, maybe it's because Broadcast doesn't expect an answer ? But I'm not sure which value it's supposed to take. Maybe it needs to be different than the other examples (publish has id=4).
Also, it is in the schema. It's the second field after schema and before description.

Reply:
I don't know if I misunderstood it, but I was referring to inserting `"id"` in the properties. We have `"$id"` after schema.
For example, this is the schema for catchup:
```json
{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "$id": "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/query/method/catchup.json",
    "description": "Match catchup on past message on a channel query",
    "type": "object",
    "additionalProperties": false,
    "properties": {
        "method": {
            "description": "[String] operation to be performed by the query",
            "const": "catchup"
        },

        "params": {
            "type": "object",
            "additionalProperties": false,
            "properties": {
                "channel": {
                    "description": "[String] name of the channel",
                    "type": "string",
                    "pattern": "^/root(/[^/]+)*$"
                }
            },

            "required": ["channel"]
        },

        "jsonrpc": {
            "$comment": "Defined by the parent, but needed here for the validation"
        },

        **"id": {
            "type": "integer"
        }**
    },

    "required": ["method", "params", "id", "jsonrpc"]
}
```
And this is the example:
```
{
    "jsonrpc": "2.0",
    "id": 3,
    "method": "catchup",
    "params": {
        "channel": "/root/p_EYbHyMv6sopI5QhEXBf40MO_eNoq7V_LygBd4c9RA="
    }
}
```

Reply:
You're right. I got confused with the different "id" fields in the schemas and what the error messages were saying ... I'll try again later today while taking into account the comments. Thank you :)

Reply:
No problem!🙂